jetty-annotations/src/test/resources/web-fragment4false.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<web-fragment 
+   xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-fragment_4_0.xsd"
+   version="4.0"> 
+  <name>ardvaark</name>
+</web-fragment>
```

Comment:
This fragment is called `...4false`, but IIUC there is a difference between not set and explicitly set to `false`.
If that's the case, do we have tests that differentiate between the two?

Reply:
Nope, missing or false are equivalent according to the spec.